test/e2e/run_volume_test.go

```diff
@@ -676,13 +676,9 @@ VOLUME /test/`, ALPINE)
 		session.WaitWithDefaultTimeout()
 		Expect(session).Should(Exit(0))
 
-		// Test overlay mount when lowerdir is relative path.
-		f, err = os.Create("hello")
-		Expect(err).ToNot(HaveOccurred(), "os.Create")
-		f.Close()
 		session = podmanTest.Podman([]string{"run", "--rm", "-v", ".:/app:O", ALPINE, "ls", "/app"})
 		session.WaitWithDefaultTimeout()
-		Expect(session.OutputToString()).To(ContainSubstring("hello"))
+		Expect(session.OutputToString()).To(ContainSubstring("run_volume_test.go"))
```

Comment:
super-duper-minor nit: instead of hardcoding, might it be possible to use `Caller()`? Again, super-minor, it seems impossible for this to fail: even if the source file is renamed, the error would be caught immediately. It's just one of my neuroses.

Reply:
Mhh searching around there is `CurrentGinkgoTestDescription().FileName`, I wasn't ware of such functionality before.
If I have to repush I can add it.